src/status_im2/common/standard_auth/view.cljs

```diff
@@ -0,0 +1,174 @@
+(ns status-im2.common.standard-auth.view
+  (:require [react-native.core :as rn]
+            [quo2.components.numbered-keyboard.numbered-keyboard.view :as numbered-keyboard]
```

Comment:
numbered keyboard should also be exported from quo.core. 
@ilmotta, @yqrashawn, @FFFra any idea if we can put some error of requires of quo library (maybe just for everything in quo.components?? 🤔 ) it would prevent us needing to comment this feedback 👍

Reply:
@J-Son89 Made the needed update and used `numbered-keyboard` from `quo`.

Reply:
Added to the lint improvement issue https://github.com/status-im/status-mobile/issues/17641

Reply:
> Added to the lint improvement issue https://github.com/status-im/status-mobile/issues/17641

Thanks @yqrashawn.

> any idea if we can put some error of requires of quo library (maybe just for everything in quo.components?? 

Yes @J-Son89, we can always at least do something similar to the script `lint-re-frame-in-quo-components.sh` in case we can't do it via clj-kondo.

There's also this interesting tool to establish verifiable boundaries between "modules" in a repo. https://github.com/fabiodomingues/clj-depend I don't know if it would work in our repo, but it's interesting nonetheless. Similar tools exist in other programming languages.